services/packagist/packagist.tester.js

```diff
@@ -34,7 +34,7 @@ t.create('gets the package version of symfony 2.8')
 
```

Comment:
Could we split this into multiple `*.tester.js` files to match the respective `*.service.js` files?